code/modules/mob/living/carbon/human/life.dm

```diff
@@ -629,6 +638,26 @@
 
 	breath.update_values()
 	return 1
+	
+/mob/living/carbon/human/proc/play_inhale(var/mob/living/M, var/exhale)
+	var/suit_inhale_sound
+	if(species.suit_inhale_sound)
+		suit_inhale_sound = species.suit_inhale_sound
+	else // Failsafe
+		suit_inhale_sound = 'sound/effects/mob_effects/suit_breathe_in.ogg'
+	
+	M << sound(suit_inhale_sound,0,0,0,100)
+	if(!exhale) // Did we fail exhale? If no, play it after inhale finishes.
+		addtimer(CALLBACK(src, .proc/play_exhale, M), 5 SECONDS)

```

Comment:
may be worth making this TIMER_STOPPABLE and holding the id so that the pending exhale can be cancelled when the mask/suit is removed

Reply:
Apologies for the super-late reply, but we do actually check if exhale can be played, hence the !exhale, and then in play_exhale, we simply play it, because timing it after the inhale was easiest, and by the time we reach this check, we've already tried to breathe (and therefore know if our breath is valid or not)

Reply:
The point was that stuff can happen in the intervening five seconds, but it's a small issue either way.